server/server.go

```diff
@@ -20,6 +21,7 @@ import (
 	"github.com/CeresDB/ceresmeta/server/id"
 	"github.com/CeresDB/ceresmeta/server/member"
 	metagrpc "github.com/CeresDB/ceresmeta/server/service/grpc"
+	http2 "github.com/CeresDB/ceresmeta/server/service/http"
```

Comment:
Why use `https`?

Reply:
It's not https, golang has already contains http package, and I define another one, import my http package will be auto renamed as http2.